crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
> It's a syncronisation problem either way.

Since reference to HGTG didn't work, let's take π. If there were two functions in different modules in same library that worked better with static value of π, would you still insist on having single copy of the variable specifically with rationale that it's synchronization problem? Yes, I'm suggesting that [in the context] list of months' abbreviations is as universal as binary value of π. "Universal" does not refer to physical universe, but to specific domain of human activities, programming contemporary computers. Which is also the reason for why π is referred to as "binary value of".